community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/executionplan/CompiledExecutionResult.scala

```diff
@@ -174,12 +175,12 @@ class CompiledExecutionResult(taskCloser: TaskCloser,
   private def props(x: PropertyContainer): String = {
     val readOperations = statement.readOperations()
     val (properties, propFcn, id) = x match {
-      case n: Node => (readOperations.nodeGetAllProperties(n.getId).asScala.map(_.propertyKeyId()), readOperations.nodeGetProperty _, n.getId )
-      case r: Relationship => (readOperations.relationshipGetAllProperties(r.getId).asScala.map(_.propertyKeyId()), readOperations.relationshipGetProperty _, r.getId)
+      case n: Node => (asScala(readOperations.nodeGetPropertyKeys(n.getId)), readOperations.nodeGetProperty _, n.getId )
+      case r: Relationship => (asScala(readOperations.relationshipGetPropertyKeys(r.getId)), readOperations.relationshipGetProperty _, r.getId)
```

Comment:
It's a PrimitiveIntIterator which doesn't have that, so need to use JavaConversionSupport instead. Unless there's some magic that I'm not aware of that makes it work anyway.